sass/MLFacet/_MLFacetValues.scss

```diff
@@ -47,3 +48,27 @@ $active-text-opacity: 0.6;
     }
   }
 }
+
+.coveo-ml-facet-show-more,
+.coveo-ml-facet-show-less {
+  width: 100%;
+  text-align: left;
+  margin-top: 2px;
+  padding: 3px 0;
+  background: none;
+  border: none;
+  font-size: $font-size-regular;
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+.coveo-ml-facet-show-more {
+  color: $color-vibrant-blue;
+}
+
+.coveo-ml-facet-show-less {
```

Comment:
I think the fact that this is gray, even if it's like that in the mockup, could potentially cause accessibility issues (that would be picked up "automatically" by the accessibilityTests), since it is a clickable element with a bad color contrast.

Reply:
When discussed with Anne Catherine, she made me change the light gray for the dark one that won't cause accessibility issues according to her.
![Screen Shot 2019-04-02 at 1 52 28 PM](https://user-images.githubusercontent.com/4923043/55424584-96f59e00-554e-11e9-9c58-a6f60d3848dd.png)